src/main/java/seedu/address/model/person/Person.java

```diff
@@ -51,6 +55,21 @@ public Address getAddress() {
         return address;
     }
 
+
+    /*
+    public Location getLocation() {
```

Comment:
You can also leave such changes on the personal branch next time as discussed in the tutorial instead of pushing it to team master branch. Remember, every version should be a better version and not include non-working functionality